packages/verkle/src/util/bytes.ts

```diff
@@ -8,15 +8,29 @@
  */
 export function matchingBytesLength(bytes1: Uint8Array, bytes2: Uint8Array): number {
   let count = 0
-
-  // The minimum length of both arrays
   const minLength = Math.min(bytes1.length, bytes2.length)
 
-  for (let i = 0; i < minLength; i++) {
+  // Unroll the loop for better performance
```

Comment:
Did you do performance testing to show that this is faster?  It's not obvious to me why this is any faster than iterating element by element.

Reply:
Yes the benchmark file is in the above comment: https://github.com/ethereumjs/ethereumjs-monorepo/pull/3441#issuecomment-2141083701. Since multiple bytes are being compared at a time, the loop runs less times and handles the remaining comparisons outside of the first loop.

Reply:
Ah, sorry, didn't see this.  Thanks for the comparison!